packages/ExternalLink/package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Don't forget `.npmignore`

Reply:
we don't usually include that in each component, can you elaborate where/why?

Reply:
Hmm I think this was discussed before at some point... But personally I don't think there's enough reason to publish things like `src` and `stories`. If people disagree we could change this, but let's do so consistently.

Reply:
Agreed.  I think most components do have the `.npmignore`, so that only the `lib` folder gets published for consumption on npm.  They should _all_ have it though.

Reply:
You can copy this one:
https://github.com/acl-services/paprika/blob/master/packages/Button/.npmignore

Reply:
Oh, It looks like each component does have a `npmignore` file, i just don't see them in my ide, i'll add one

Reply:
FYI there is also #88 - for later.

Reply:
Ok, ive added `.npmignore` for this component as well as in `Checkbox`, `Dropdown`, `Confirmation` components